src/features/query-bar/__tests__/FilterButton-test.js

```diff
@@ -221,20 +223,41 @@ describe('feature/query-bar/components/filter/FilterButton', () => {
         const validConditions = [{ values: [1] }];
         const incompleteConditions = [{ values: [] }];
 
-        test.each`
-            description                                                            | transientConditions     | stateProperty            | expectedState
-            ${'Should empty out transientConditions when Apply button is clicked'} | ${validConditions}      | ${'transientConditions'} | ${[]}
-            ${'Should close the menu when Apply Button is clicked'}                | ${validConditions}      | ${'isMenuOpen'}          | ${false}
-            ${'Should show error message if not all conditions are valid'}         | ${incompleteConditions} | ${'areErrorsEnabled'}    | ${true}
-        `('$description', ({ transientConditions, stateProperty, expectedState }) => {
+        test('Should empty out transientConditions when Apply button is clicked', () => {
             const wrapper = getWrapper({ conditions: [] });
             wrapper.instance().setState({
-                transientConditions,
+                transientConditions: validConditions,
+                isMenuOpen: true,
+            });
+
+            wrapper.find('PrimaryButton').simulate('click');
+
+            expect(wrapper.state('transientConditions')).toEqual([]);
+        });
+
+        test('Should close the menu when Apply Button is clicked', () => {
+            const wrapper = getWrapper({ conditions: [] });
+            wrapper.instance().setState({
+                transientConditions: validConditions,
+                isMenuOpen: true,
+            });
+
+            wrapper.find('PrimaryButton').simulate('click');
+
+            const Flyout = wrapper.find('Flyout');
+            expect(Flyout.props().overlayIsVisible).toBe(false);
+        });
+
+        test('Should show error message if not all conditions are valid', () => {
+            const wrapper = getWrapper({ conditions: [] });
+            wrapper.instance().setState({
+                transientConditions: incompleteConditions,
+                isMenuOpen: true,
             });
 
-            wrapper.instance().applyFilters();
+            wrapper.find('PrimaryButton').simulate('click');
 
-            expect(wrapper.state(stateProperty)).toEqual(expectedState);
+            expect(wrapper.state('areErrorsEnabled')).toEqual(true);
```

Comment:
The unit test description says one thing but line 260 does another. Is it possible to verify the error message is present?